content/en/infrastructure/resource_catalog/_index.md

```diff
@@ -102,7 +116,8 @@ Click the **Share** button and select **Share event** to share a link to the cur
 [3]: /integrations/#cat-notification
 [4]: /account_management/teams
 [5]: https://app.datadoghq.com/infrastructure/catalog/configuration
-
-
+[6]: /integrations/amazon_config/#events
+[7]: https://docs.datadoghq.com/help/
+[8]: #
```

Comment:
Is this a placeholder?

Reply:
@estherk15 yep, thanks for reminding me about that. It's meant to link to a section of another page that doesn't exist yet, but I now have approval on a separate PR to add that section, so I've updated this in [90f5c10](https://github.com/DataDog/documentation/pull/23681/commits/90f5c10f7a8947139e2b0a29a1525c4dbc425b78)